account.go

```diff
@@ -1,22 +1,260 @@
 package stripe
 
-import "encoding/json"
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+)
+
+type LegalEntityType string
+
+type IdentityVerificationStatus string
+
+type Interval string
+
+const (
+	Individual  LegalEntityType = "individual"
+	SoleProp    LegalEntityType = "sole_prop"
+	LLC         LegalEntityType = "llc"
+	Partnership LegalEntityType = "partnership"
+	Corp        LegalEntityType = "corporation"
+
+	IdentityVerificationWaiting  IdentityVerificationStatus = "waiting"
+	IdentityVerificationPending  IdentityVerificationStatus = "pending"
+	IdentityVerificationVerified IdentityVerificationStatus = "verified"
+	IdentityVerificationFailed   IdentityVerificationStatus = "failed"
+
+	Manual Interval = "manual"
+	Day    Interval = "daily"
+	Week   Interval = "weekly"
+	Month  Interval = "monthly"
+)
+
+type AccountParams struct {
+	Params
+	Country, Email, DefaultCurrency, Statement, BusinessName, SupportPhone string
+	LegalEntity                                                            *LegalEntity
+	Managed                                                                bool
+}
+
+type AccountListParams struct {
+	ListParams
+}
 
 // Account is the resource representing youe Stripe account.
 // For more details see https://stripe.com/docs/api/#account.
 type Account struct {
-	ID            string `json:"id"`
-	ChargeEnabled bool   `json:"charge_enabled"`
-	Country       string `json:"country"`
+	ID             string `json:"id"`
+	ChargesEnabled bool   `json:"charges_enabled"`
+	Country        string `json:"country"`
 	// Currencies is the list of supported currencies.
 	Currencies       []string `json:"currencies_supported"`
 	DefaultCurrency  string   `json:"default_currency"`
 	DetailsSubmitted bool     `json:"details_submitted"`
-	TransferEnabled  bool     `json:"transfer_enabled"`
+	TransfersEnabled bool     `json:"transfers_enabled"`
 	Name             string   `json:"display_name"`
 	Email            string   `json:"email"`
 	Statement        string   `json:"statement_descriptor"`
 	Timezone         string   `json:"timezone"`
+	BusinessName     string   `json:"business_name"`
+	SupportPhone     string   `json:"support_phone"`
+	ProductDesc      string   `json:"product_description"`
+	Managed          bool     `json:"managed"`
+	Keys             *struct {
+		Secret  string `json:"secret"`
+		Publish string `json:"publishable"`
+	} `json:"keys"`
+	Verification *struct {
+		Fields    []string `json:"fields_needed"`
+		Due       *int64   `json:"due_by"`
+		Contacted bool     `json:"contacted"`
+	} `json:"verification"`
+	LegalEntity      *LegalEntity      `json:"legal_entity"`
+	TransferSchedule *TransferSchedule `json:"transfer_schedule"`
```

Comment:
https://qa.stripe.com/docs/connect/accounts has some of it. We're pushing most doc changes into the new format for now to avoid needing to double-write.